src/components/EditSVGPage.tsx

```diff
@@ -147,6 +152,9 @@ const EditSVGPage = ({ svgString, setLoading }: Props) => {
         id="myCanvas"
       ></canvas>
       <div className="float-right">
+        <Button style={{ marginRight: 5 }} onClick={onClearClicked}>
```

Comment:
You should put the button on the left side of the edit window and keep save on the right, potentially use a flex box and justify space between